src/components/UserInfo/UserInfo.tsx

```diff
@@ -2,0 +1,17 @@
+import React from 'react';
+
+import './UserInfo.scss';
+
+import { User } from '../../types';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  return (
+    <a className="UserInfo" href={`mailto:${user?.email}`}>
+      {user?.name}
```

Comment:
[CODE KNOWLEDGE] - Similarly, you do not need to use the optional chaining operator `?.` when accessing `user.name`. Since `user` is required and of type `User`, you can directly access `user.name` without checking for its existence.